orchagent/qosorch.cpp

```diff
@@ -2088,6 +2088,24 @@ task_process_status QosOrch::handlePortQosMapTable(Consumer& consumer, KeyOpFiel
                         return task_process_status::task_invalid_entry;
                     }
                 }
+                if (attr.id == SAI_PORT_ATTR_QOS_TC_AND_COLOR_TO_DSCP_MAP) {
```

Comment:
Since this attribute is set above, and if implemention doesn't support setting, looks like its failing and returning from above check itself. can you please confirm?

Reply:
Yes @prsunny If the set_attribute API for SAI_PORT_ATTR_QOS_TC_AND_COLOR_TO_DSCP_MAP fails, below code of setting the new SAI attribute(UPDATE_DSCP) will not be invoked.

Reply:
then why do we need to check capability here?

Reply:
Some vendor SAI/SDK may not support this additional attribute(SAI_PORT_ATTR_UPDATE_DSCP) to update the DSCP.

Reply:
in that case, it would be anyways failing in the previous` if` condition, right?

Reply:
ok